src/Workspaces/Remote/ServiceHub/Services/ServiceBase.cs

```diff
@@ -37,6 +37,13 @@ internal abstract class ServiceBase : IDisposable
         protected readonly TraceSource Logger;
         protected readonly AssetStorage AssetStorage;
 
+        private readonly RemoteWorkspaceManager _workspaceManager;
+
+        internal static readonly RemoteWorkspaceManager s_defaultWorkspaceManager = new RemoteWorkspaceManager();
```

Comment:
can you doc the distinctino between these?  i.e. when is the default one used and when is the non-default one used?

Reply:
Sure, the non-default is only used in tests.

Reply:
ah ok.  def worth doc'ing that :)